### PR TITLE
merged all html files into the doc file and deleted unwanted html files

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**

### CODE REVIEW

File: CSS/avatar.css [hunk 1]
```diff
@@ -1,26 +1,52 @@
...
+
+.avatar-lgr {
+    height: 8rem;
+    width: 8rem;
 }
 
+
+/* 
+
+
```
Comment (harshitadaswani) [inline CSS/avatar.css]: remove empty space

File: CSS/avatar.css [hunk 2]
```diff
@@ -1,26 +1,52 @@
...
     border-radius: 0.5rem;
-    border: 1px solid var(--dark-primary-text);
-    background-color: var(--primary-bg-color);
-    box-shadow: 1px 2px 2px 1px var(--dark-primary-text);
+    /* background-color: var(--primary-bg-color); */
+    box-shadow: 1px 1px 2px 2px var(--dark-primary-text);
+}
+
+.avatar-images {
+    /* background-image: url("../images/tungsten3.jpg"); */
```
Comment (harshitadaswani) [inline CSS/avatar.css]: remove unnecessary comments

File: CSS/base/nav.css [hunk 3]
```diff
@@ -1,136 +1,60 @@
...
-    justify-content: center;
-    width: 66%;
-    transform: translateY(0%);
-    transition: transform 0.5s ease-in;
 }
 
-.nav-links a {
+.navlink a {
+    margin-right: 1.5rem;
+    padding: 0.3rem 0.6rem 0.3rem 0.6rem;
```
Comment (harshitadaswani) [inline CSS/base/nav.css]: use shorthand property `padding: 0.3rem 0.6rem`

File: CSS/base/reset.css [hunk 4]
```diff
@@ -94,21 +94,22 @@ button {
...
     font-size: 1.3rem;
     margin: 0.5rem 0rem 0.5rem 0rem;
 }
 
 .paragraphs {
     font-size: 1.3rem;
-    width: 70%;
+    width: 90%;
     text-align: center;
     margin: 1rem 0rem 1rem 0rem;
```
Comment (harshitadaswani) [inline CSS/base/reset.css]: use shorthand property

File: CSS/base/reset.css [hunk 5]
```diff
@@ -94,21 +94,22 @@ button {
...
     font-weight: bold;
     border: 1px solid var(--dark-primary-text);
     box-shadow: 1px 1px 2px 2px var(--dark-primary-text);
     padding: 0.5rem 1rem 0.5rem 1rem;
```
Comment (harshitadaswani) [inline CSS/base/reset.css]: again shorthand property

File: CSS/base/reset.css [hunk 6]
```diff
@@ -261,7 +254,7 @@ li {
 .iframe-parent_class {
     overflow: hidden;
     width: 101% !important;
-    max-height: 35vh !important;
+    max-height: 40vh !important;
```
Comment (harshitadaswani) [inline CSS/base/reset.css]: don't use `!important` in component library

File: main/doc.css [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+.sidebar-link {
+    display: flex;
+    flex-direction: column;
+}
+
+.sidebar-link li {
+    margin-top: 0.2rem;
+    color: var(--white);
+    padding: 0.5rem 2rem 0.5rem 2rem;
```
Comment (harshitadaswani) [inline main/doc.css]: shorthand property

File: main/doc.css [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+.main-container {
+    margin-top: 3rem;
+    width: 100vw;
+    height: calc(100vh - 3.5rem);
+    overflow: auto;
+    /* display: flex;
+    flex-direction: column;
+    align-items: center; */
```
Comment (harshitadaswani) [inline main/doc.css]: remove comments

**Review by harshitadaswani — CHANGES_REQUESTED** [review]
- use shorthand properties
- don't use `!important` in CL
- and try to make different PRs as it is difficult to review so many files together

**Review by JantuDeb — APPROVED** [review]
Implement the changes @harshitadaswani pointed out other than that everything is fine

**Pull request merged**